src/test/java/com/hotels/hcommon/ssh/SshSettingsTest.java

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (C) 2018 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.hcommon.ssh;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+
+import java.io.File;
+import java.util.Arrays;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.rules.TemporaryFolder;
+
+import com.jcraft.jsch.JSch;
+import com.jcraft.jsch.KeyPair;
+
+import com.hotels.hcommon.ssh.SshSettings;
+
+public class SshSettingsTest {
+
+  private static final String KNOWN_HOSTS = "knownHosts";
+  private static final String IDENTITY_KEY_1 = "K1";
+  private static final String IDENTITY_KEY_2 = "K2";
+
+  public @Rule ExpectedException expectedEx = ExpectedException.none();
```

Comment:
expectedEx -> expectedException (as opposed to expectedExWife or some such ;P )

Reply:
ok